src/FakeItEasy/ExceptionThrowerConfigurationExtensions.cs

```diff
@@ -130,6 +134,11 @@ public static IAfterCallSpecifiedConfiguration Throws<T1>(this IExceptionThrower
                 });
         }
 
+#if DISABLED
+
+        // This method already has a generic parameter, and it can't be inferred; if we add another, we make it unusable,
+        // especially since the new one is something unintelligible like IAnyCallConfigurationWithReturnTypeSpecificied<Foo>...
+

```

Comment:
> Ah, so I just got lucky by using this particular case as a test. Too bad.

No, it works fine if the method is in the interface. I guess we could do it like this in this particular case.

In fact, migrating all the extension methods to the interface they extend would solve all the usability problems, but it feels wrong somehow... Interfaces should be as small as possible; the current approach of having only general purpose methods in the interface and extension methods for specific use cases is clean and elegant.